From ac97849c9b0cb5a76c24f43eea519815eeaf9739 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Mon, 7 Apr 2014 12:09:38 +0200
Subject: [PATCH 2/3] Drop installation of connman configuration file

* should be done within the package management system as part of the connman-conf package

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 CMakeLists.txt       | 3 ---
 files/conf/main.conf | 3 ---
 2 files changed, 6 deletions(-)
 delete mode 100644 files/conf/main.conf

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8516103..a4c0997 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,6 +104,3 @@ webos_build_daemon()
 webos_build_system_bus_files()
 
 webos_config_build_doxygen(doc Doxyfile)
-
-# Install connman configuration file to set priorities between wired and wifi interfaces
-install(FILES files/conf/main.conf DESTINATION ${WEBOS_INSTALL_SYSCONFDIR}/connman)
diff --git a/files/conf/main.conf b/files/conf/main.conf
deleted file mode 100644
index bce852a..0000000
--- a/files/conf/main.conf
+++ /dev/null
@@ -1,3 +0,0 @@
-[General]
-DefaultAutoConnectTechnologies=ethernet,wifi
-PreferredTechnologies=ethernet,wifi
-- 
2.1.0

